.github/workflows/validate.yml

```diff
@@ -15,6 +15,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Since there are no Git actions performed in this workflow, the default token should suffice. Or is there something I missed in the inner workings of Vocab?

Reply:
Honestly trying anything at this point, but yeah there's no need for this token here.